lib/storage/src/content_manager/toc/transfer.rs

```diff
@@ -31,6 +32,7 @@ impl ShardTransferDispatcher {
     }
 }
 
+#[async_trait]
```

Comment:
Ha, neat! I never realized you don't need `#[async_trait]`, if all async methods have default impls.

Reply:
Yeah it's a bit weird to be fair. I hope some upcoming Rust release will allow us to remove it entirely.